README.md

```diff
@@ -4,17 +4,86 @@
 
 This repository contains code and tools for working with the Elastic APM AWS Lambda extension.
 
+Elastic APM Agents can be used with AWS Lambda to monitor the execution of your AWS Lambda functions.
+Currently the Node.js, Python and Java Agents are supported.
+
+The extension listens for data once per function invocation and forwards data to an APM Server.
+
 ## Documentation
 
 Ready to use Elastic APM to monitor your Lambda functions? See [Monitoring AWS Lambda Functions](https://www.elastic.co/guide/en/apm/guide/current/monitoring-aws-lambda.html) to get started.
 
 Backport any changes made in the `docs` directory to the branches that you'd like to see the docs show up in. Branch names must follow the `major.minor` naming scheme and match a new or recent Elastic Stack version.
 
-## License
-
-Apache 2.0.
-
 ## Getting Help
 
 If you find a bug, please [report an issue](https://github.com/elastic/apm-agent-go/issues).
 For any other assistance, please open or add to a topic on the [APM discuss forum](https://discuss.elastic.co/c/apm).
+
+## Contributing
+
+See [contributing](CONTRIBUTING.md) for details about reporting bugs, requesting features, or code contributions.
+
+## Testing unreleased versions
```

Comment:
Most of this is developer-oriented, and not relevant to end users. Although we have end-user docs, I think it would still be nice to keep this landing page minimal.

What do you think about moving all of the below with the exception of License and Release to a new TESTING.md doc? License should stay in this doc. And maybe link to RELEASE.md from CONTRIBUTING.md?

Reply:
Good catch! I agree, that's a lot cleaner!

> And maybe link to RELEASE.md from CONTRIBUTING.md?

I'm not sure about this. Release is already linked in the readme and won't provide much value to contributors as it's not something they would worry about. Maybe we should link to TESTING.md ? WDYT ?